src/index.html

```diff
@@ -1,12 +1,401 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a class="logo-link" href="index.html">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav-menu">
+        <ul class="nav-menu__list">
+          <li class="nav-menu__item">
+            <a class="nav-menu__link is-active" href="index.html">Apple</a>
+          </li>
+
+          <li class="nav-menu__item">
+            <a class="nav-menu__link" href="index.html">Samsung</a>
+          </li>
+
+          <li class="nav-menu__item">
+            <a class="nav-menu__link" href="index.html">Smartphones</a>
+          </li>
+
+          <li class="nav-menu__item">
+            <a 
+              class="nav-menu__link" 
+              data-qa="nav-hover" 
+              href="index.html"
+            >Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav-menu__item">
+            <a class="nav-menu__link" href="index.html">Gadgets</a>
+          </li>
+
+          <li class="nav-menu__item">
+            <a class="nav-menu__link" href="index.html">Tablets</a>
+          </li>
+
+          <li class="nav-menu__item">
+            <a class="nav-menu__link" href="index.html">Photo</a>
+          </li>
+          
+          <li class="nav-menu__item">
+            <a class="nav-menu__link" href="index.html">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-section">
+      <div class="container">
```

Comment:
why do you need this container for product card when you already have 'div' for product card
Try to avoid using divs if it not necessary
here you can create section with class 'catalog' for example, to keep all product cards inside